galaxy_ng/locale/nl/LC_MESSAGES/django.po

```diff
@@ -1,272 +1,333 @@
+"collections that require it."
+msgstr "Collectie {namespace}.{name} kon niet worden verwijderd omdat er andere collecties zijn die het nodig hebben."
+
+#: app/api/v3/viewsets/collection.py:200
+#, python-brace-format
+msgid ""
+"Collection version {namespace}.{name} {version} could not be deleted because "
+"there are other collections that require it."
+msgstr "Collectieversie {namespace}.{name} {version} kon niet worden verwijderd omdat er andere collecties zijn die het nodig hebben."
 
-#: galaxy_ng/app/api/v3/viewsets/collection.py:172
+#: app/api/v3/viewsets/collection.py:292
 #, python-format
 msgid "Path does not match: \"%s\""
-msgstr ""
+msgstr "Pad komt niet overeen: \"%s\""
 
-#: galaxy_ng/app/api/v3/viewsets/collection.py:186
+#: app/api/v3/viewsets/collection.py:313
 #, python-brace-format
 msgid "Namespace \"{0}\" does not exist."
-msgstr ""
+msgstr "Namespace \"{0}\" bestaat niet."
 
-#: galaxy_ng/app/api/v3/viewsets/collection.py:279
+#: app/api/v3/viewsets/collection.py:401
 #, python-format
 msgid "Unexpected response from content app. Code: %s."
-msgstr ""
+msgstr "Onverwachte reactie van inhoud app. Code: %s."
 
-#: galaxy_ng/app/api/v3/viewsets/collection.py:301
+#: app/api/v3/viewsets/collection.py:431
 #, python-format
 msgid "Collection %s not found"
-msgstr ""
+msgstr "Collectie %s niet gevonden"
 
-#: galaxy_ng/app/api/v3/viewsets/collection.py:309
+#: app/api/v3/viewsets/collection.py:439
 #, python-format
 msgid "Repo(s) for moving collection %s not found"
-msgstr ""
+msgstr "Repo('s) voor verplaatsen collectie %s niet gevonden"
 
-#: galaxy_ng/app/api/v3/viewsets/collection.py:313
+#: app/api/v3/viewsets/collection.py:443
 #, python-format
 msgid "Collection %s not found in source repo"
-msgstr ""
+msgstr "Collectie %s niet gevonden in bron-repo"
 
-#: galaxy_ng/app/api/v3/viewsets/collection.py:317
+#: app/api/v3/viewsets/collection.py:447
 #, python-format
 msgid "Collection %s already found in destination repo"
-msgstr ""
+msgstr "Collectie %s reeds gevonden in doel-repo"
 
-#: galaxy_ng/app/api/v3/viewsets/namespace.py:61
+#: app/api/v3/viewsets/namespace.py:67
 #, python-format
 msgid "A namespace named %s already exists."
-msgstr ""
+msgstr "Een namespace met de naam %s bestaat al."
+
+#: app/api/v3/viewsets/namespace.py:88
+#, python-brace-format
+msgid ""
+"Namespace {name} cannot be deleted because there are still collections "
+"associated with it."
+msgstr "Namespace {name} kan niet worden verwijderd omdat er nog collecties aan zijn gekoppeld."
+
+#: app/auth/keycloak.py:51
+msgid "Authentication failed."
+msgstr "Authenticatie mislukt."
 
-#: galaxy_ng/app/exceptions.py:8
+#: app/exceptions.py:8
 msgid "Data conflicts with existing entity."
-msgstr ""
+msgstr "Gegevens conflicteren met bestaande entiteit."
+
+#: app/tasks/index_registry.py:26
+#, python-brace-format
+msgid "Failed to create container {remote_name}. {error}"
+msgstr "Kan container {remote_name}. {error} niet aanmaken"
```

Comment:
this should probably say 

```suggestion
msgstr "Kan container {remote_name} niet aanmaken. {error}"
```

Reply:
I agreed.  I added your change.  To make sure this gets translated properly for future releases, I also commented on that string in memsource.  This will ensure that the Translation Memory in Memsource is correct (which pre-fills many translations).  
![image](https://user-images.githubusercontent.com/11698892/158486068-c1f426b2-64ae-45cc-ba27-f72d77c6b161.png)